tests/test_utils.py

```diff
@@ -46,8 +46,11 @@ def remove_schema(sender, json=None, record=None, **kwargs):
         if '$schema' in json:
             del json['$schema']
 
-    models_committed.connect(process_models_committed_signal, sender=app)
-
+    with app.app_context():
+        # NOTE: We have to use `db.get_app()` because Flask-SQLAlchemy v2.2
+        # changed the way it picks a sender for the model signals.
+        models_committed.connect(process_models_committed_signal,
+                                 sender=db.get_app())
```

Comment:
Will this work with previous Flask-SQLAlchemy versions as well? 

Alternatively, we can simply remove the use of ``model_commited`` signal since this is anyway bad pattern to use to get records indexed (it's too generic to really be useful).